src/OpenTelemetry/Trace/Configuration/TracerFactory.cs

```diff
@@ -82,6 +82,7 @@ private TracerFactory(TracerBuilder builder)
         /// Creates tracerSdk factory.
         /// </summary>
         /// <param name="configure">Function that configures tracerSdk factory.</param>
+        /// <returns><see cref="TracerFactory"/>.</returns>
```

Comment:
Cn you please make the message more human friendly.

Reply:
I tried to update based on the context that I have and based on other comments that we had in the code. Let me know if I need to change something else!

Reply:
looks good